comlink.ts

```diff
@@ -28,16 +28,44 @@ export interface Endpoint {
 // To avoid Promise<Promise<T>>
 type Promisify<T> = T extends Promise<any> ? T : Promise<T>;
 
-// ProxiedObject<T> is equivalent to T, except that all properties are now promises and
-// all functions now return promises. It effectively async-ifies an object.
+/**
+ * Symbol that gets added to objects by `Comlink.proxy()`.
+ */
+export const proxyValueSymbol = Symbol("comlinkProxyValue");
```

Comment:
Why did you `export` this one?

Reply:
Because it's now part of the public API. For types, you can get it anyway through `keyof ProxyValue` if you want to, and for the value you can create a proxied value then save it from there.

Besides that, I see a use case of implementing classes this way:
```ts
class WorkerService implements ProxyValue {
  [proxyValueSymbol] = true;

  // ...
}
```

Which would make sure instances of this class are always proxied.